_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -25,19 +25,19 @@ This rule applies to any HTML element that is [visible][] and has a [declared va
 - [word-spacing][],
 - [letter-spacing][],
 - [line-height][];
-  which are set by the `style` attribute, and have the [important flag][].
+  which are set by the `style` attribute.
 
 ## Expectation 1
 
-The test target's [computed][] [word-spacing][] is at least `0.16` times it's [computed][] [font-size][].
+The test target's [computed][] [word-spacing][] either does not have the [important flag][] or is at least `0.16` times its [computed][] [font-size][].
```

Comment:
Doesn't the important flag relate to the declared value, not the computed one?
```suggestion
For the test target's [word-spacing][] property, either it has no [declared value][], or its [declared value][] does not have the [important flag][] set, or its [computed value]][] is at least `0.16` times the [computed value][] of the test target [font-size][] property.
```
(repeat or the other two expectations)